docs/src/main/sphinx/connector/redshift.md

```diff
@@ -64,6 +64,31 @@ documentation](https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-configura
 ```{include} jdbc-authentication.fragment
 ```
 
+### UNLOAD configuration
+
+The connector brings parallelism by converting Trino query to corresponding 
+Redshift `UNLOAD` command to unload query results on to the S3 bucket in the 
+form of Parquet files and reading these Parquet files from S3 bucket. Parquet 
+files will be removed as Trino query finishes. However, it's advisable to 
+define a custom life cycle policy on the S3 bucket used for unloading the 
+Redshift query results.
+This feature intends to bring query results faster compared to the default JDBC 
+approach when fetching large query result set from Redshift.
+One of the limitation is that Redshift cluster and S3 must be in the same AWS 
+region.
```

Comment:
This feature is supported only when the Redshift cluster and the configured S3 bucket are in the same AWS region.